### PR TITLE
feat: Adding MongoDB container to Robotoff in dev

### PR DESCRIPTION
**Pull request by Jagrutiti**
### What
I have tried to add the MongoDB container to docker.
When I run `docker-compose up` no error is thrown. So I think it works fine.
Screenshot from 2022-04-09 18-11-04

But when I call the `save_annotation` function through pdb it throws the following error. 
Screenshot from 2022-04-09 18-18-14

Before adding these new changes the error message was like so

Screenshot from 2022-04-08 19-55-47

### Closes
- #692
### Part of 
- I was actually trying to work on https://github.com/openfoodfacts/robotoff/issues/606 when I encountered this error.

### CODE REVIEW

**Review by alexgarel — CHANGES_REQUESTED** [review]
You don't need the mongodb.yml file

**Comment by Jagrutiti** [comment]
Hey @alexgarel 

I am just not sure how to provide a demo database to the user to download and extract?

File: docker/dev.yml [hunk 1]
```diff
@@ -44,6 +44,12 @@ services:
   scheduler:
     <<: *networks-productopener-local
     <<: *robotoff-dev
+  mongodb:
+    # Note: keep it as close a possible to mongodb.yml
```
Comment (raphael0202) [inline docker/dev.yml:48]: This line should be removed I think

File: doc/how-to-guides/deployment/dev-install.md [hunk 2]
```diff
@@ -51,4 +51,37 @@ After cloning the repository:
...
+    ```
+    tar xzf off-dev-mongo-dump.tar.gz
+    docker cp -a dump robotoff_mongodb_1:/var/tmp/
+    docker exec mongodb mongorestore /var/tmp/dump
+
+    ```
+
+    First we have extracted the zip file. Next we copy the dump in our running MongoDB service in Robotoff `robotoff_mongodb_1`. Lastly, we restore the dump from Robotoff's service to `mongodb` container.
+
+    The MongoDB configuration is now compelete.
```
Comment (raphael0202) [inline doc/how-to-guides/deployment/dev-install.md:77]: compelete -> complete

**Review by raphael0202 — APPROVED** [review]

**Pull request merged**